libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md

```diff
@@ -39,6 +39,163 @@ The Telem1 & Telem2 port has RTS/CTS pins, the other UARTs do not have RTS/CTS.
  -All JST-GH 6 and 4 pin connectors with FMUv6 standard pinout
  -USB-C
 
+### TELEM1 
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin </th>
+   <th>Signal </th>
+   <th>Volt </th>
+   </tr>
+   <tr>
+   <td>1 (red)</td>
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2 (blk)</td>
+   <td>TX (OUT)</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>3 (blk)</td>
+   <td>RX (IN)</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>4 (blk)</td>
+   <td>CTS</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>5 (blk)</td>
+   <td>RTS</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>6 (blk)</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+
+### GPS + RCIN(UART6)
```

Comment:
this is not correct, right? no I2C for compass, UART6 RX is not on the connector

Reply:
Hey Henry, just checked. We have I2C only on power ports. This connector has UART6 or RCIN.

Reply:
unfortunate....so external GPS/compass modules (the typical kind) cant connect to the GPS connector....so only CAN GPS/Compass can be used (using internal compass is usually too prone to motor interference)....
in any case the GPS/RC connector show TX1/TX2 and RX1/RX2 not UART6 and UART7(GPS)

Reply:
Yes. Most of our customers use CAN RM3100 for magnetometer. And I2C is there on Power ports for smart battery SMBus few our customers use.

Reply:
again, GPS is TX7/RX7 and RC is RX6....that is not what is listed below